### PR TITLE
Fix/docker issues

### PR DESCRIPTION
**Pull request by noahschechter**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .dockerignore
-.env
```
Comment (wazam) [inline .dockerignore:2]: if we build image with private .env's, we must remember to not push that image to public repos

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+if [ ! -f .env ]
```
Comment (wazam) [inline entrypoint.sh:1]: i like this file

**Review by wazam — APPROVED** [review]
i am signed off

File: .dockerignore (hunk 1 above)
Comment (noahschechter) [reply .dockerignore:2]: Yep. I think we can fix that by not doing COPY . . and replace that with something more explicit

**Pull request merged**